Sources/DeepLearning/MNISTdataset.swift

```diff
@@ -0,0 +1,91 @@
+import Python
+import TensorFlow
+
+let np = Python.import("numpy")
```

Comment:
Do you think you could do a native-swift implementation, instead of falling back to Python?

Reply:
I am not sure if there is any native function in Foundation library to decompress gz files which is the file hosted in MNIST official URL. I can do a pure swift implementation for opening the files and converting the file into swift array. Just that I may have to use a third party framework for extraction process.